Microsoft.Recognizers.Text.NumberWithUnit/Extractors/NumberWithUnitExtractor.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Recognizers.Text.NumberWithUnit
+{
+    public class NumberWithUnitExtractor : IExtractor
+    {
+        private readonly INumberWithUnitExtractorConfiguration config;
+
+        private readonly HashSet<Regex> SuffixRegexes = new HashSet<Regex>();
+        private readonly HashSet<Regex> PrefixRegexes = new HashSet<Regex>();
+
+        private readonly int _maxPrefixMatchLen;
+        
+        public NumberWithUnitExtractor(INumberWithUnitExtractorConfiguration config)
```

Comment:
This looks to be an issue throughout this code base, but public interfaces, classes and non-override methods really should have XMLDocs describing them. I would not hold this code review, but we should file a work item to fix this.